Plugin/src/main/java/xyz/kyngs/librelogin/common/config/ConfigurationKeys.java

```diff
@@ -358,4 +358,12 @@ This includes (but is not limited to):
                     """,
             ConfigurateHelper::getBoolean
     );
+    public static final ConfigurationKey<String> LIMBO_PORT_RANGE = new ConfigurationKey<>(
+      "limbo-port-range",
+      "30000-40000",
+            """
+                    !!THIS OPTION IS IRRELEVANT WHEN USING PAPER!! Defines port(s) that limbo server can be bounded to.
+                    """ ,
```

Comment:
Why's this a text block? Considering the comment only has a single line, I think it would be better if it were a normal string.

Reply:
Yeah, you're right